Makefile

```diff
@@ -499,39 +514,70 @@ genconfig:
 	source setenv.bash && \
 	(cd src/github.com/getlantern/flashlight/genconfig && ./genconfig.bash)
 
-android-lib: docker-mobile
-	@source setenv.bash && \
-	cd $(LANTERN_MOBILE_DIR)
-	@$(call docker-up) && \
-	$(DOCKER) run -v $$PWD/src:/src $(DOCKER_MOBILE_IMAGE_TAG) /bin/bash -c \ "cd /src/github.com/getlantern/lantern-mobile && gomobile bind -target=android -o=$(LANTERN_MOBILE_LIBRARY) -ldflags="$(LDFLAGS) $$EXTRA_LDFLAGS" ." && \
-	if [ -d "$(FIRETWEET_MAIN_DIR)" ]; then \
-		cp -v $(LANTERN_MOBILE_DIR)/$(LANTERN_MOBILE_LIBRARY) $(FIRETWEET_MAIN_DIR)/libs/$(LANTERN_MOBILE_LIBRARY); \
-	else \
-		echo ""; \
-		echo "Either no FIRETWEET_MAIN_DIR variable was passed or the given value is not a";\
-		echo "directory. You'll have to copy the $(LANTERN_MOBILE_LIBRARY) file manually:"; \
-		echo ""; \
-		echo "cp -v $(LANTERN_MOBILE_DIR)/$(LANTERN_MOBILE_LIBRARY) \$$FIRETWEET_MAIN_DIR"; \
-	fi
+$(ANDROID_LIB):
+	source setenv.bash && \
+	go install golang.org/x/mobile/cmd/gomobile && \
+	gomobile init && \
```

Comment:
You're right, it does take a while.  I've updated the makefile so that it only installs and initializes gomobile when necessary.